Packs/GCP-IAM/Integrations/GCP-IAM/GCP-IAM.py

```diff
@@ -0,0 +1,2432 @@
+                        f'If you wish to add a new policy, consider using the {command_name} command.')
+
+    return iam_policy
+
+
+def add_members_to_policy(role: str, iam_policy: list, members: list, command_name: str) -> list:
+    """
+    Append members to policy role members.
+    Args:
+        role (str): The name of policy role.
+        iam_policy (list): IAM policies.
+        members (list): Members to append to policy.
+        command_name (str): An alternative  command that will be displayed to the user in case of an error.
+
+    Returns:
+        list: Updated policies.
+
+    """
+    role_found = False
+    for policy in iam_policy:
+        if policy.get('role') == role:
+            policy['members'].extend(members)
+            role_found = True
+            break
+
+    if not role_found:
+        raise Exception('The provided role is not part of the project IAM policies.'
+                        f'If you wish to add a new policy, consider using the {command_name} command.')
+
+    return iam_policy
+
+
+def generate_service_account_command_output(response: dict, output_key: str = None,
+                                            readable_header: str = 'Service account information:') -> CommandResults:
+    """
+    Generate command output for service account commands.
+    Args:
+        response (dict): API response from GCP.
+        output_key (str): Used to access to required data in the response.
+        readable_header (str): Readable message header for XSOAR war room.
+
+    Returns:
+        CommandResults: outputs, readable outputs and raw response for XSOAR.
+
+    """
+    if output_key:
+        outputs = copy.deepcopy(response.get(output_key, []))
+    else:
+        outputs = copy.deepcopy(response)
+
+    if not isinstance(outputs, list):
+        outputs = [outputs]
+
+    for output in outputs:
+        output["disabled"] = output.get("disabled", False)
+
+    readable_output = tableToMarkdown(readable_header,
+                                      outputs,
+                                      headers=["name", "displayName", "description", "projectId"],
+                                      headerTransform=pascalToSpace
+                                      )
+
+    command_results = CommandResults(
+        readable_output=readable_output,
+        outputs_prefix='GCP.IAM.ServiceAccount',
+        outputs_key_field='name',
+        outputs=outputs,
+        raw_response=response
+    )
+
+    return command_results
+
+
+def gcp_iam_projects_get_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """
+    List projects under the specified parent, or retrieve specific project information.
+    Args:
+        client (Client): GCP API client.
+        args (dict): Command arguments from XSOAR.
+
+    Returns:
+        CommandResults: outputs, readable outputs and raw response for XSOAR.
+
+    """
+
+    project_name = args.get('project_name')
```

Comment:
we should support providing a comma-separated list of project names to retrieve.